ledfx/virtuals.py

```diff
@@ -395,8 +400,6 @@ def assemble_frame(self):
             self._transition_effect is not None
             and self._transition_effect.is_active
             and hasattr(self._transition_effect, "pixels")
-            and self._config["transition_mode"] != "None"
```

Comment:
These are removed to prevent an in flight transition getting orphaned, as they reference _config, which can change async. Instead we never create a transition in these circumstances